modules/dublincore/src/main/java/org/opencastproject/metadata/dublincore/StaticMetadataServiceDublinCoreImpl.java

```diff
@@ -124,16 +123,11 @@ public void activate(@SuppressWarnings("rawtypes") Map properties) {
    */
   @Override
   public StaticMetadata getMetadata(final MediaPackage mp) {
-    return mlist(list(mp.getCatalogs(DublinCoreCatalog.ANY_DUBLINCORE)))
-            .find(flavorPredicate(MediaPackageElements.EPISODE))
-            .flatMap(loader)
-            .map(new Function<DublinCoreCatalog, StaticMetadata>() {
-              @Override
-              public StaticMetadata apply(DublinCoreCatalog episode) {
-                return newStaticMetadataFromEpisode(episode);
-              }
-            })
-            .getOrElse((StaticMetadata) null);
+    Catalog[] catalogs = mp.getCatalogs(MediaPackageElements.EPISODE);
+    if (catalogs.length > 0) {
+      return newStaticMetadataFromEpisode(DublinCoreUtil.loadDublinCore(workspace, catalogs[0]));
```

Comment:
This seems like it's making the assumption that the *first* catalog is *always* the episode catalog.  Is that a safe assumption?